src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,264 @@
+splash screen or displays a loading indicator, this moment is not very relevant
+to the user.
+
+In the past we've recommended performance metrics like [First Meaningful Paint
+(FMP)](https://web.dev/first-meaningful-paint/) and [Speed Index
+(SI)](https://web.dev/speed-index/) (both available in Lighthouse) to help
+capture more of the loading experience after the initial paint, but these
+metrics are complex, hard to explain, and often wrong&mdash;meaning they still do
+not identify when the main content of the page has loaded.
+
+Sometimes simpler is better, and the latest research done in the [Web
+Performance Working Group](https://www.w3.org/webperf/) suggests that a more
+accurate way to measure when the main content of a page is loaded is to look at
+when the largest element was rendered.
+
+## Largest Contentful Paint defined
+
+The [Largest Contentful Paint
+(LCP)](https://wicg.github.io/largest-contentful-paint/) API, available in
+Chrome 77 beta, reports the render time of the largest content element visible
+in the viewport.
+
+### What elements are considered?
+
+As currently specified, the types of elements considered for Largest Contentful
+Paint are:
+
+* `<img>` elements
+* `<image>` elements inside an `<svg>` element
+* Any element with a background image that is fetched (as opposed to a gradient)
+* Block-level elements containing text nodes or other inline-level text elements
+  children.
+
+Note, restricting the elements considered to this limited set was intentional
+in order to keep things simple in the beginning. Additional elements (e.g.
+`<svg>`, `<video>`) may be added in the future as more research is conducted.
+
+### How is an element's size determined?
+
+The size of the element reported for Largest Contentful Paint is the size
+that's visible to the user within the viewport. If the element extends outside
+of the viewport, or if any of the element is clipped or has non-visible
+[overflow](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow), those
+portions do not count toward the element's size.
+
+For image elements, only the visible size of the rendered image is considered.
+The original size of the image is not factored into the element size
+calculation, nor is any margin, padding, or border that is applied to the image
+via CSS.
+
+For text elements, only the size of its text nodes is considered (the smallest
+rectangle that encompases all text nodes). As with images, the element's
+margin, padding, and border applied via CSS are not considered part of its
+contentful size.
+
+{% Aside %}
+  Determining which text nodes belong to which elements can sometimes
+  be tricky, especially for elements whose children includes inline elements and
+  plain text nodes but also block-level elements. The key point is that every
+  text node belongs to (and only to) its closest block-level ancestor element.
+  In spec terms: each text nodes belong to the element that generates its
```

Comment:
Change 'text nodes belong' to 'text node belongs'.

Reply:
Done.